das_api/src/api/mod.rs

```diff
@@ -74,6 +74,7 @@ pub struct SearchAssets {
     pub royalty_target: Option<String>,
     pub royalty_amount: Option<u32>,
     pub burnt: Option<bool>,
+    pub json_uri: Option<String>,
```

Comment:
Nice catch, will fix

Reply:
imo, ideally array params should be disabled, since it’s error prone and causes compatibility problems, so prob we need api v2 when we want to add more param